.github/workflows/translate.yml

```diff
@@ -83,3 +84,45 @@ jobs:
               --which_modules=DynCore \
               --threshold_overrides_file=./tests/savepoint/translate/overrides/standard.yaml \
               ./tests/savepoint
+        - name: Orchestrated dace:cpu FvTp2d
+          run: |
+            rm -rf .gt_cache_FV3_A
+            export FV3_DACEMODE=BuildAndRun
+            export PACE_FLOAT_PRECISION=64
+            export PACE_TEST_N_THRESHOLD_SAMPLES=0
+            export OMP_NUM_THREADS=10
+            export PACE_LOGLEVEL=Debug
+            pytest \
+              -v -s --data_path=./test_data/8.1.3/c12_6ranks_standard/dycore \
+              --backend=dace:cpu \
+              --which_modules=FvTp2d \
+              --threshold_overrides_file=./tests/savepoint/translate/overrides/standard.yaml \
+              ./tests/savepoint
+        - name: Orchestrated dace:cpu D_SW
+          run: |
+            rm -rf .gt_cache_FV3_A
+            export FV3_DACEMODE=BuildAndRun
+            export PACE_FLOAT_PRECISION=64
+            export PACE_TEST_N_THRESHOLD_SAMPLES=0
+            export OMP_NUM_THREADS=10
+            export PACE_LOGLEVEL=Debug
+            pytest \
+              -v -s --data_path=./test_data/8.1.3/c12_6ranks_standard/dycore \
+              --backend=dace:cpu \
+              --which_modules=D_SW \
+              --threshold_overrides_file=./tests/savepoint/translate/overrides/standard.yaml \
+              ./tests/savepoint
+        - name: Orchestrated dace:cpu Remapping
```

Comment:
This is the _bare_ minimum for orchestration. We should probably run some pySHiELD code and the full FVDynamics case as well

Reply:
As of  commit `453e6b7` the omp threads have been rolled back to 1, the NDSL version is back at 2024.04.00, and added `coverage` to the test suite. I timed the current test suite to take 5601.23 s (~93.35 mins). Would it make sense to add in the pySHiELD code and FVDynamics case in another PR?

Reply:
93 mins and we are only touching 33% of the code? That's... disappointing. Is the coverage report aggregating everything correctly since all the test are done one by one?

Reply:
OpenMP won't help, the execution is not slow part, the build is. Frank restricted to one rank test (e.g. testing the data generated on rank 0 instead of the entire cube) and that should take care of the slowness of the test overall